<ui version="4.0" >
 <class>pskForm</class>
 <widget class="QDialog" name="pskForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>299</width>
    <height>144</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>299</width>
    <height>144</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>299</width>
    <height>144</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Credentials Required</string>
  </property>
  <property name="windowIcon" >
   <iconset>../../../../SEA_Ant/ui/Skins/Default/icons/prod_color.png</iconset>
  </property>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>60</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Preshared Key :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="disconnectBtn" >
   <property name="geometry" >
    <rect>
     <x>210</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Disconnect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OkayBtn" >
   <property name="geometry" >
    <rect>
     <x>120</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Okay</string>
   </property>
   <property name="default" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="pskEdit" >
   <property name="geometry" >
    <rect>
     <x>100</x>
     <y>60</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode" >
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>20</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Connection : </string>
   </property>
  </widget>
  <widget class="QLabel" name="pskConnName" >
   <property name="geometry" >
    <rect>
     <x>100</x>
     <y>20</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Unknown</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>pskEdit</tabstop>
  <tabstop>OkayBtn</tabstop>
  <tabstop>disconnectBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
